medibloc/chain.json

```
@@ -0,0 +1,79 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "panacea",
+    "status": "live",
+    "network_type": "mainnet",
+    "pretty_name": "Panacea",
```

```suggestion
    "pretty_name": "MediBloc",
```
I think the name `MediBloc` is more comprehensive for users who are already familiar with CEXs.

---

Ah.. Let's go with the `Panacea` as you wrote first because MediBloc might launch multiple chains in the future.